### ISSUE TITLE
Error starting the server

### ISSUE THREAD

**Issue by MarcoAlbaETT** [sci-wms/sci-wms#147]
Hi!

when I try to execute "docker-compose up -d" the worker fails with this error:
```
web_1     | Sourcing profile...
web_1     | Migrating sci-wms...
web_1     | Traceback (most recent call last):
web_1     |   File "manage.py", line 27, in <module>
web_1     |     execute_from_command_line(sys.argv)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
web_1     |     utility.execute()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/core/management/__init__.py", line 357, in execute
web_1     |     django.setup()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/__init__.py", line 24, in setup
web_1     |     apps.populate(settings.INSTALLED_APPS)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/apps/registry.py", line 91, in populate
web_1     |     app_config = AppConfig.create(entry)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/apps/config.py", line 90, in create
web_1     |     module = import_module(entry)
web_1     |   File "/opt/conda/lib/python3.7/importlib/__init__.py", line 127, in import_module
web_1     |     return _bootstrap._gcd_import(name[level:], package, level)
web_1     |   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
web_1     |   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
web_1     |   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
web_1     |   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
web_1     |   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
web_1     |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/contrib/djhuey/__init__.py", line 100, in <module>
web_1     |     HUEY = backend_cls(name, **huey_config)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 111, in __init__
web_1     |     self.storage = self.create_storage()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 128, in create_storage
web_1     |     return self.get_storage(**self.storage_kwargs)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 140, in get_storage
web_1     |     return Storage(self.name, **kwargs)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 697, in __init__
web_1     |     super(SqliteStorage, self).__init__(name)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 625, in __init__
web_1     |     self.initialize_schema()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 661, in initialize_schema
web_1     |     with self.db(commit=True, close=True) as curs:
web_1     |   File "/opt/conda/lib/python3.7/contextlib.py", line 112, in __enter__
web_1     |     return next(self.gen)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 644, in db
web_1     |     conn = self.conn
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 636, in conn
web_1     |     self._state.set_connection(self._create_connection())
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 701, in _create_connection
web_1     |     **self._conn_kwargs)
web_1     | TypeError: 'blocking' is an invalid keyword argument for this function
web_1     | Collecting sci-wms static assets...
web_1     | Traceback (most recent call last):
web_1     |   File "manage.py", line 27, in <module>
web_1     |     execute_from_command_line(sys.argv)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
web_1     |     utility.execute()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/core/management/__init__.py", line 357, in execute
web_1     |     django.setup()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/__init__.py", line 24, in setup
web_1     |     apps.populate(settings.INSTALLED_APPS)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/apps/registry.py", line 91, in populate
web_1     |     app_config = AppConfig.create(entry)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/apps/config.py", line 90, in create
web_1     |     module = import_module(entry)
web_1     |   File "/opt/conda/lib/python3.7/importlib/__init__.py", line 127, in import_module
web_1     |     return _bootstrap._gcd_import(name[level:], package, level)
web_1     |   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
web_1     |   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
web_1     |   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
web_1     |   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
web_1     |   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
web_1     |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/contrib/djhuey/__init__.py", line 100, in <module>
web_1     |     HUEY = backend_cls(name, **huey_config)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 111, in __init__
web_1     |     self.storage = self.create_storage()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 128, in create_storage
web_1     |     return self.get_storage(**self.storage_kwargs)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 140, in get_storage
web_1     |     return Storage(self.name, **kwargs)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 697, in __init__
web_1     |     super(SqliteStorage, self).__init__(name)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 625, in __init__
web_1     |     self.initialize_schema()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 661, in initialize_schema
web_1     |     with self.db(commit=True, close=True) as curs:
web_1     |   File "/opt/conda/lib/python3.7/contextlib.py", line 112, in __enter__
web_1     |     return next(self.gen)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 644, in db
web_1     |     conn = self.conn
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 636, in conn
web_1     |     self._state.set_connection(self._create_connection())
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 701, in _create_connection
web_1     |     **self._conn_kwargs)
web_1     | TypeError: 'blocking' is an invalid keyword argument for this function
web_1     | ========================================================================
web_1     | sci-wms user:         "sciwmsuser"
web_1     | sci-wms password:     "yuE83yO1YuYa4eyC"
web_1     | ========================================================================
web_1     | Starting sci-wms...
web_1     | [2020-08-25 08:53:25 +0000] [36] [INFO] Starting gunicorn 20.0.4
web_1     | [2020-08-25 08:53:25 +0000] [36] [INFO] Listening at: http://0.0.0.0:7002 (36)
web_1     | [2020-08-25 08:53:25 +0000] [36] [INFO] Using worker: eventlet
web_1     | [2020-08-25 08:53:25 +0000] [40] [INFO] Booting worker with pid: 40
web_1     | [2020-08-25 08:53:25 +0000] [40] [ERROR] Exception in worker process
web_1     | Traceback (most recent call last):
web_1     |   File "/opt/conda/lib/python3.7/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
web_1     |     worker.init_process()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/gunicorn/workers/geventlet.py", line 99, in init_process
web_1     |     super().init_process()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/gunicorn/workers/base.py", line 119, in init_process
web_1     |     self.load_wsgi()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/gunicorn/workers/base.py", line 144, in load_wsgi
web_1     |     self.wsgi = self.app.wsgi()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
web_1     |     self.callable = self.load()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load
web_1     |     return self.load_wsgiapp()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
web_1     |     return util.import_app(self.app_uri)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/gunicorn/util.py", line 358, in import_app
web_1     |     mod = importlib.import_module(module)
web_1     |   File "/opt/conda/lib/python3.7/importlib/__init__.py", line 127, in import_module
web_1     |     return _bootstrap._gcd_import(name[level:], package, level)
web_1     |   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
web_1     |   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
web_1     |   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
web_1     |   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
web_1     |   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
web_1     |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
web_1     |   File "/srv/sci-wms/sciwms/wsgi.py", line 4, in <module>
web_1     |     application = Cling(get_wsgi_application())
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
web_1     |     django.setup(set_prefix=False)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/__init__.py", line 24, in setup
web_1     |     apps.populate(settings.INSTALLED_APPS)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/apps/registry.py", line 91, in populate
web_1     |     app_config = AppConfig.create(entry)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/django/apps/config.py", line 90, in create
web_1     |     module = import_module(entry)
web_1     |   File "/opt/conda/lib/python3.7/importlib/__init__.py", line 127, in import_module
web_1     |     return _bootstrap._gcd_import(name[level:], package, level)
web_1     |   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
web_1     |   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
web_1     |   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
web_1     |   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
web_1     |   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
web_1     |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/contrib/djhuey/__init__.py", line 100, in <module>
web_1     |     HUEY = backend_cls(name, **huey_config)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 111, in __init__
web_1     |     self.storage = self.create_storage()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 128, in create_storage
web_1     |     return self.get_storage(**self.storage_kwargs)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 140, in get_storage
web_1     |     return Storage(self.name, **kwargs)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 697, in __init__
web_1     |     super(SqliteStorage, self).__init__(name)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 625, in __init__
web_1     |     self.initialize_schema()
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 661, in initialize_schema
web_1     |     with self.db(commit=True, close=True) as curs:
web_1     |   File "/opt/conda/lib/python3.7/contextlib.py", line 112, in __enter__
web_1     |     return next(self.gen)
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 644, in db
web_1     |     conn = self.conn
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 636, in conn
web_1     |     self._state.set_connection(self._create_connection())
web_1     |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 701, in _create_connection
web_1     |     **self._conn_kwargs)
web_1     | TypeError: 'blocking' is an invalid keyword argument for this function
web_1     | [2020-08-25 08:53:25 +0000] [40] [INFO] Worker exiting (pid: 40)
web_1     | [2020-08-25 08:53:25 +0000] [36] [INFO] Shutting down: Master
web_1     | [2020-08-25 08:53:25 +0000] [36] [INFO] Reason: Worker failed to boot.
worker_1  | Traceback (most recent call last):
worker_1  |   File "manage.py", line 27, in <module>
worker_1  |     execute_from_command_line(sys.argv)
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
worker_1  |     utility.execute()
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/django/core/management/__init__.py", line 357, in execute
worker_1  |     django.setup()
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/django/__init__.py", line 24, in setup
worker_1  |     apps.populate(settings.INSTALLED_APPS)
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/django/apps/registry.py", line 91, in populate
worker_1  |     app_config = AppConfig.create(entry)
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/django/apps/config.py", line 90, in create
worker_1  |     module = import_module(entry)
worker_1  |   File "/opt/conda/lib/python3.7/importlib/__init__.py", line 127, in import_module
worker_1  |     return _bootstrap._gcd_import(name[level:], package, level)
worker_1  |   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
worker_1  |   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
worker_1  |   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
worker_1  |   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
worker_1  |   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
worker_1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/contrib/djhuey/__init__.py", line 100, in <module>
worker_1  |     HUEY = backend_cls(name, **huey_config)
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 111, in __init__
worker_1  |     self.storage = self.create_storage()
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 128, in create_storage
worker_1  |     return self.get_storage(**self.storage_kwargs)
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/api.py", line 140, in get_storage
worker_1  |     return Storage(self.name, **kwargs)
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 697, in __init__
worker_1  |     super(SqliteStorage, self).__init__(name)
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 625, in __init__
worker_1  |     self.initialize_schema()
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 661, in initialize_schema
worker_1  |     with self.db(commit=True, close=True) as curs:
worker_1  |   File "/opt/conda/lib/python3.7/contextlib.py", line 112, in __enter__
worker_1  |     return next(self.gen)
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 644, in db
worker_1  |     conn = self.conn
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 636, in conn
worker_1  |     self._state.set_connection(self._create_connection())
worker_1  |   File "/opt/conda/lib/python3.7/site-packages/huey/storage.py", line 701, in _create_connection
worker_1  |     **self._conn_kwargs)
worker_1  | TypeError: 'blocking' is an invalid keyword argument for this function
```

I've installed docker(Docker version 19.03.12, build 48a66213fe) and docker-compose (docker-compose version 1.26.2, build eefe0d31) on CentOS 7.

I've used the following docker-compose.yml:
```
version: '3.2'

services:

  web:
    image: axiom/sci-wms
    environment:
      DJANGO_SETTINGS_MODULE: sciwms.settings.quick
    ports:
      - "7002:7002"
    volumes:
      - topologydata:/srv/sci-wms/wms/topology:ro
      - dbdata:/srv/sci-wms/sciwms/db/

  worker:
    image: axiom/sci-wms
    environment:
      DJANGO_SETTINGS_MODULE: sciwms.settings.quick
    volumes:
      - topologydata:/srv/sci-wms/wms/topology
      - dbdata:/srv/sci-wms/sciwms/db/
    command: python manage.py run_huey

volumes:
  topologydata:
  dbdata:
```

What I'm missing?

thank you!

Marco

**Comment by kwilcox** [comment]
@MarcoAlbaETT I've fixed the problem in https://github.com/sci-wms/sci-wms/commit/e870a2bbb9d5eee1a4bc9e76434ca94cc86a1b5e, could you please `docker pull axiom/sci-wms:latest` and try again? If it works for you I will tag a new version.

**Comment by MarcoAlbaETT** [comment]
It's working!

Marco

**Comment by kwilcox** [comment]
Great! I've just released `2.0.1`, once the build finishes you should update your docker-compose file to use the specific version to prevent future updates from creeping into your deploy. Good luck!
